bolt_hole[6] bolt_holes
rack_hole[12] rack_holes
